### ISSUE TITLE
Ruby 2.7 warnings

### ISSUE THREAD

**Issue by somepaulo** [wpscanteam/wpscan#1448]
Before submitting an issue, please make sure you fully read any potential error messages output and did some research on your own.

### Subject of the issue
The bundled Ruby 2.7 is throwing multiple errors during scans.

### Your environment
* Version of WPScan: 3.7.5
* Version of Ruby: 2.7
* Operating System (OS): Arch Linux

### Steps to reproduce
Run any scan.

### Expected behavior
No Ruby errors or warnings should appear.

### Actual behavior
Output throws three different warnings.

Two at the start of the scan:
/opt/wpscan/vendor/bundle/ruby/2.7.0/gems/public_suffix-4.0.1/lib/public_suffix/list.rb:51: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/opt/wpscan/vendor/bundle/ruby/2.7.0/gems/public_suffix-4.0.1/lib/public_suffix/list.rb:69: warning: The called method `parse' is defined here

And many instances of a third warning at different points of the scan:
/opt/wpscan/vendor/bundle/ruby/2.7.0/gems/cms_scanner-0.7.1/lib/cms_scanner/web_site.rb:39: warning: URI.escape is obsolete

### What have you already tried
Reinstalling wpscan and Ruby.

Things you have tried (where relevant):

* Update WPScan to the latest version [y]
* Update Ruby to the latest version [y]
* Ensure you can reach the target site using cURL [y]
* Proxied WPScan through a HTTP proxy to view the raw traffic [n]
* Ensure you are using a supported Operating System (Linux and macOS) [y]

**Comment by erwanlr** [comment]
v3.7.6 fixes the warning related to WPScan. The one from `public_suffix` seem to have been fixed in 4.0.3 (https://github.com/weppos/publicsuffix-ruby/commit/bac3a1d05c26e5fc1a8dd7919fad4d8a4ca39163)

So a `gem update` should fix all that

**Comment by foozzi** [comment]
> v3.7.6 fixes the warning related to WPScan. The one from `public_suffix` seem to have been fixed in 4.0.3 ([weppos/publicsuffix-ruby@bac3a1d](https://github.com/weppos/publicsuffix-ruby/commit/bac3a1d05c26e5fc1a8dd7919fad4d8a4ca39163))
> 
> So a `gem update` should fix all that

patched https://github.com/weppos/publicsuffix-ruby/commit/bac3a1d05c26e5fc1a8dd7919fad4d8a4ca39163
Warnings have not disappeared

**Comment by erwanlr** [comment]
Was not able to reproduce:

```
$ ruby -v
ruby 2.7.0p0 (2019-12-25 revision 647ee6f091) [x86_64-darwin18]
```

```
$ gem list wpscan cms_scanner public_suffix

*** LOCAL GEMS ***

wpscan (3.7.6)

*** LOCAL GEMS ***

cms_scanner (0.8.1)

*** LOCAL GEMS ***

public_suffix (4.0.3)
```

```
$ wpscan --url google.com
_______________________________________________________________
         __          _______   _____
         \ \        / /  __ \ / ____|
          \ \  /\  / /| |__) | (___   ___  __ _ _ __ ®
           \ \/  \/ / |  ___/ \___ \ / __|/ _` | '_ \
            \  /\  /  | |     ____) | (__| (_| | | | |
             \/  \/   |_|    |_____/ \___|\__,_|_| |_|

         WordPress Security Scanner by the WPScan Team
                         Version 3.7.6
       Sponsored by Automattic - https://automattic.com/
       @_WPScan_, @ethicalhack3r, @erwan_lr, @firefart
_______________________________________________________________


Scan Aborted: The URL supplied redirects to http://www.google.com/. Use the --ignore-main-redirect option to ignore the redirection and scan the target, or change the --url option value to the redirected URL.
```
